.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
 before_install:
   - bash scripts/setup-osx.sh
 install:
-  - python3 scripts/compile_lang.py
```

Comment:
Was there a reason for this change? I'm good with it, but curious what's the purpose?

Reply:
I rebased my commits on top of `scripts/compile_lang.py` `import config`. I think I should re-rebase my commits then :).

Reply:
Hrm... that might actually be what I needed to get the config to work in the script.will test it out tonight